tests/expectations/metrics/test_core.py

```diff
@@ -386,7 +386,7 @@ def test_map_of_type_sa(sa):
     desired_metric = MetricConfiguration(
         metric_name="table.column_types",
         metric_domain_kwargs={},
-        metric_value_kwargs={},
+        metric_value_kwargs=dict(),
     )
```

Comment:
Why not leave this as `{}`? Isn't it faster?

Reply:
Plus consistent with `metric_domain_kwargs`!

Reply:
@NathanFarmer Good point.  The `dict()` pattern was established last winter when this new V3 Expectations was created.  We can do a clean-up PR (after a couple of feature-based PRs) and make all usage in  `MetricConfiguration ` adhere to `{}` consistently.  Thanks.